media-gfx/blender/patches/blender-2.79b.patchset

```diff
@@ -891,5 +929,55 @@ index b123d50..7f2fac9 100644
  
  	if (!ibuf)
 -- 
-2.21.0
+2.24.1
+
+From 2888d696409b685e22c195914d91338c4bcec086 Mon Sep 17 00:00:00 2001
+From: X512 <danger_mail@list.ru>
+Date: Wed, 29 Jan 2020 14:07:31 +0900
+Subject: [PATCH 09/10] Haiku: fix build for 32 bit
```

Comment:
Should we try to upstream this? All other patches already upstreamed already afaik.

Reply:
It is OK to upstream last 2 commits, it fixes 32 bit build. This fixes are only applied to 2.79b version, it are unlikely required for 2.8+.

Reply:
Let's wait till we get 2.8 then.

Reply:
Blender 2.81 is now working: https://discuss.haiku-os.org/t/blender-2-81-is-working-on-haiku/9235.